packages/protocol/test/common/onboarding.ts

```diff
@@ -0,0 +1,424 @@
+import { getSignatureForMetaTransaction, MetaTransaction } from '@celo/protocol/lib/meta-tx-utils'
+import { CeloContractName } from '@celo/protocol/lib/registry-utils'
+import { getParsedSignatureOfAddress } from '@celo/protocol/lib/signing-utils'
+import {
+  assertEqualBN,
+  assumeOwnership,
+  getDerivedKey,
+  getVerificationCodeSignature,
+  KeyOffsets,
+  unlockAndAuthorizeKey,
+} from '@celo/protocol/lib/test-utils'
+import { getDeployedProxiedContract } from '@celo/protocol/lib/web3-utils'
+import { ensureLeading0x, privateKeyToAddress, trimLeading0x } from '@celo/utils/lib/address'
+import {
+  AccountsContract,
+  AccountsInstance,
+  AttestationsTestContract,
+  AttestationsTestInstance,
+  MetaTransactionWalletContract,
+  MetaTransactionWalletDeployerContract,
+  MetaTransactionWalletDeployerInstance,
+  MetaTransactionWalletInstance,
+  MockRandomContract,
+  MockRandomInstance,
+  MockValidatorsContract,
+  ProxyCloneFactoryContract,
+  ProxyCloneFactoryInstance,
+  ProxyContract,
+  ProxyInstance,
+  RegistryInstance,
+  StableTokenInstance,
+} from 'types'
+
+const executeMetaTransaction = async (
+  signer: string,
+  mtw: MetaTransactionWalletInstance,
+  metaTx: MetaTransaction
+) => {
+  const { v, r, s } = await getSignatureForMetaTransaction(signer, mtw.address, metaTx)
+  return mtw.executeMetaTransaction(metaTx.destination, metaTx.value, metaTx.data, v, r, s)
+}
+
+const getExecuteMetaTransactionData = async (
+  signer: string,
+  mtw: MetaTransactionWalletInstance,
+  metaTx: MetaTransaction
+) => {
+  const { v, r, s } = await getSignatureForMetaTransaction(signer, mtw.address, metaTx)
+  // @ts-ignore
+  return mtw.contract.methods
+    .executeMetaTransaction(metaTx.destination, metaTx.value, metaTx.data, v, r, s)
+    .encodeABI()
+}
+
+const logCosts = false
```

Comment:
Is there a nice way to toggle this with command line flags? Leaving this setting as a variable to manually change feels a little dirty.

Reply:
Given the desire to ship this in the next wallet release I would propose that this is an acceptable shortcut.